### PR TITLE
Maui ebook

### PR DESCRIPTION
**Pull request by michaelstonis**
## Summary

Setup of TOC and index page for maui ebook. This is still a WIP and will need links and images added.

### CODE REVIEW

**Comment by nishanil** [comment]
@IEvangelist Are you ok with this?

**Review by IEvangelist — APPROVED** [review]

**Comment by michaelstonis** [comment]
> @michaelstonis Can you change the folder name to be just "maui" instead of "enterprise-application-patterns-using-maui"

done.

File: docs/architecture/enterprise-application-patterns-using-maui/index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+All other marks and logos are property of their respective owners.
+
+Authors:
+
+> **[Michael Stonis](https://github.com/michaelstonis)**, Mobile Software Architect, Eightbot.com
+
+> **[David Britch](https://github.com/davidbritch)**, Developer/Writer, Microsoft Corp.
+
+> **[Javier Suarez Ruiz](https://github.com/jsuarezruiz)**, Developer/Writer, Microsoft Corp.
+
```
[the remark this answers is not in the document]
Comment (nishanil) [reply docs/architecture/enterprise-application-patterns-using-maui/index.md]: @michaelstonis Can you add @IEvangelist to the reviewers' section, please?

**Comment by nishanil** [comment]
@IEvangelist We are good with the changes. Should we worry about the build warnings now or later when we make final merge to main?

**Review by nishanil — APPROVED** [review]

File: docs/architecture/maui/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+This book focuses on building cross-platform enterprise apps using .NET MAUI. As such, it should be read in its entirety to provide a foundation of understanding such apps and their technical considerations. The book, along with its sample app, can also serve as a starting point or reference for creating a new enterprise app. Use the associated sample app as a template for the new app, or to see how to organize an app's component parts. Then, refer back to this guide for architectural guidance. You can find the sample app on [GitHub](https://github.com/dotnet-architecture/eshop-mobile-client).
+
+## What this book doesn't cover
+
+This book is aimed at readers who are already familiar with .NET MAUI. It does cover some concepts of .NET MAUI to help better illustrate the topic, but it does not cover most controls and concepts in any detail. For general guidance on building a new .NET MAUI app, please refer to the [Building your first app](/dotnet/maui/get-started/first-app) guide in the .NET MAUI Documentation
+
+### Additional resources
+
+You can find the official .NET MAUI home page and documentation at <https://docs.microsoft.com/en-us/dotnet/maui/>.
```
Comment (IEvangelist) [inline docs/architecture/maui/index.md:77]:
```suggestion
You can find the official .NET MAUI home page and documentation at <https://docs.microsoft.com/dotnet/maui/>.
```

File: docs/architecture/maui/introduction.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Introduction
```
Comment (IEvangelist) [inline docs/architecture/maui/introduction.md:2]: Introduction to .NET MAUI

**Review by IEvangelist — APPROVED** [review]

File: docs/architecture/maui/introduction.md (hunk 3 above)
Comment (IEvangelist) [inline docs/architecture/maui/introduction.md:2]:
```suggestion
title: Introduction to .NET MAUI
```

**Pull request merged**